docs/configuration-examples.md

```diff
@@ -0,0 +1,97 @@
+# Configuration examples
+
+Examples of configuration TOML files, you can start with one of this and modify to meet your needs.
+
+## [rest.bitcoin.com tech stack](./architecture#1.-rest.bitcoin.com-tech-stack)
+
+```toml
+# Example rest.bitcoin.com tech stack TOML config file
+
+network = "mainnet"
+rest_service = "bchrest"
+exposed_ports = true
+docker_network = "bch-devsuite-network"
+
+[local_node]
+node = "bu"
+wait_time = 6
+
+[slp]
+username = "actorforth"
+password = "123ldsfoijqwerj"
+
+[bch_rpc_conf]
+host = "http://bch-node"
+port = "8332"
+username = "actorforth"
+password = "BWrzap0bqMjezoeHtOzgOOcUgzkxaL6w"
+```
+
+---
+
+## [Chris Troutner JS tech stack](./architecture#2.-chris-troutner-js-tech-stack)
+
+```toml
+# Example Chris Troutner JS Tech Stack TOML config file
+
+network = "mainnet"
+rest_service = "bchapi"
+exposed_ports = true
+docker_network = "bch-devsuite-network"
+
+[local_node]
+node = "bu"
+wait_time = 6
```

Comment:
This seems redundant to this https://github.com/ActorForth/bch-devsuite/pull/51/files#diff-cc74d50f1cf8bad20bd110a302a8b4f95245cdca769e1aa8a28e4890f5a4adb9R69